eks-distro-base/check_update.sh

```diff
@@ -21,21 +21,21 @@ set -x
 SCRIPT_ROOT="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd -P)"
 
 IMAGE_TAG=$1
+PACKAGE_LIST=$2
 
 BASE_IMAGE=public.ecr.aws/eks-distro-build-tooling/eks-distro-base:$(cat ${SCRIPT_ROOT}/../EKS_DISTRO_BASE_TAG_FILE)
-mkdir check-update
-cat << 'EOF' >> check-update/Dockerfile
```

Comment:
When I used this without the quotes, the heredoc took the `echo $?` literally instead of resolving it to the exit code.  So the return_value file had `$?` instead of the actual exit code of `yum check_update`.